waiter/src/waiter/service_description.clj

```diff
@@ -498,10 +506,24 @@
   (-> (token->token-data kv-store token token-data-keys false include-deleted)
       token-data->token-description))
 
+(defn token->service-parameter-template
+  "Retrieves the service description template for the given token containing only the service parameters."
```

Comment:
What's the difference between a service description template and a service parameter template?

Reply:
Service description == service parameters + system generated metadata